onnx/test/checker_test.py

```diff
@@ -1025,6 +1026,22 @@ def test_loop_with_same_initializer_input_above_ir4(self) -> None:
             shape_inference.InferenceError, checker.check_model, model, True
         )
 
+    def test_empty_list_attribute(self):
+        model = onnx.parser.parse_model(
+            """
+            <
+                ir_version: 7,
+                opset_import: [ "" : 17]
+            >
+            agraph (float[N] x) => (int64[M] y)
+            {
+                y = Constant <value_ints: ints = []>()
```

Comment:
Add another test case for floats since empty list is now allowed for floats as well.

Reply:
Done